tests/cpp/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # Copyright (c) 2013-2015, EPFL/Blue Brain Project
 #                     Daniel Nachbaur <daniel.nachbaur@epfl.ch>
 #                     Raphael Dumusc <raphael.dumusc@epfl.ch>
+#
+# Change this number when adding tests to force a CMake run: 0
```

Comment:
I never understood how this is useful.. running cmake yourself is simpler than thinking about changing a random line in this file...